src/pages/instances/actions/PublishInstanceBtn.tsx

```diff
@@ -0,0 +1,47 @@
+import { FC } from "react";
+import { LxdInstance } from "types/instance";
+import { ActionButton } from "@canonical/react-components";
+import usePortal from "react-useportal";
+import CreateImageFromInstanceForm from "../forms/CreateImageFromInstanceForm";
+
+interface Props {
+  instance: LxdInstance;
+  isDeleting: boolean;
+  isStopping: boolean;
+}
+
+const PublishInstanceBtn: FC<Props> = ({
```

Comment:
let's also rename this component as well as file name i.e. `CreateInstanceFrom...`